docs/flashbots-protect/rpc/cancellations.md

```diff
@@ -2,7 +2,7 @@
 title: canceling transactions from being submitted further
 ---
 
-Transactions that are submitted to Flashbots Protect RPC (*not* using fast mode) are submitted to the Flashbots Relay for the next 25 blocks. Once a transaction is submitted from the relay to a miner we cannot "recall" it. However, we can cancel submitting transactions for future blocks.
+Transactions that are submitted to Flashbots Protect RPC are submitted to block builders for up to 3 days. Once a transaction is included in a block and submitted to validators we cannot "recall" it. However, we can prevent a transaction from being included in future blocks.
```

Comment:
A better phrasing is it will stay in the blockbuilder's private mempool for up to 3 days before being dropped.